web_dashboard_open_action/static/src/xml/web_dashboard_open_action.xml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
While adding xml declarations is a good idea, preferably on ports it would be better if we kept the diff minimal.